applications/opencost/2.2.7/helmrelease/cm.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: opencost-2.2.7-config-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
+    priorityClassName: dkp-high-priority
+    service:
+      enabled: true
+      labels:
+        prometheus.kommander.d2iq.io/select: "true"
+        servicemonitor.kommander.mesosphere.io/path: "metrics"
+        servicemonitor.kommander.mesosphere.io/port: "http"
+    opencost:
+      selectorLabels:
+        servicemonitor.kommander.mesosphere.io/path: "metrics"
+      prometheus:
+        internal:
+          enabled: true
+          namespaceName: ${releaseNamespace}
+          serviceName: kube-prometheus-stack-prometheus
+          port: 9090
+
+      dataRetention:
+        dailyResolutionDays: 14
+
+      # frontend
+      ui:
+        image:
+          registry: ghcr.io
+          repository: opencost/opencost-ui
+          tag: "1.117.3"
+        enabled: true
+        ingress:
+          enabled: true
+          ingressClassName: "kommander-traefik"
+          annotations:
+            traefik.ingress.kubernetes.io/router.tls: "true"
+            traefik.ingress.kubernetes.io/router.middlewares: "${workspaceNamespace}-stripprefixes@kubernetescrd,${workspaceNamespace}-forwardauth@kubernetescrd"
+          hosts:
+            - host: ""
+              paths:
+                - /dkp/opencost/frontend
+
+      # backend
+      exporter:
+        image:
+          registry: ghcr.io
+          repository: opencost/opencost
+          tag: "1.117.3"
+        extraEnv:
+          CURRENT_CLUSTER_ID_FILTER_ENABLED: "false"
```

Comment:
I think for single cluster opencost, this should be `true` or maybe even omitted? (we'd set it for centralized opencost)

Reply:
omitted